Update Kyma tutorials (#63532)
sap-tutorials/btp-onboarding: tutorials/btp-kyma-onb-01-env/btp-kyma-onb-01-env.md

## You will learn

  - How to create a subaccount for enterprise accounts or enter a trial subaccount
  - How to entitle your subaccount to use Kyma runtime
  - Enable Kyma runtime

## Prerequisites

An SAP BTP enterprise account with unused Kyma entitlement or an SAP BTP trial account.

## Intro

> **NOTE:** If you follow this tutorial as part of a mission from SAP Discovery Center, consider using the Quick Account Setup instead as it completes the steps outlined here for you. However, if you prefer to do the steps manually, this tutorial is for you.

[SAP BTP, Kyma runtime](https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-environment?version=Cloud) provides a fully managed Kubernetes runtime based on the open-source project [Kyma](https://kyma-project.io/#/). With this cloud-native solution, developers can extend SAP solutions with serverless Functions and combine them with containerized microservices.

### Create a subaccount for Kyma runtime

You can use an existing subaccount of your SAP BTP global enterprise account and enable Kyma, or create a new subaccount for this tutorial.

In this step, you create a new subaccount for your Kyma runtime.

> > **NOTE:** In an SAP BTP **trial account**, a subaccount with Kyma entitlement is already preconfigured. You could create another subaccount in trial, but you it will not have unused entitlements to use Kyma. You need to delete the Kyma entitlement in the preconfigured subaccount first.

Author: I would choose "it" instead of "you".

1. Navigate to your Global Account home page. Select **Create > Subaccount**.

   ![Create a new subaccount](images/22_1_createsub.png)

2. Enter the **Display Name** and your preferred **Region**.

   For more details about Regions where Kyma is available, see [Discovery Center](https://discovery-center.cloud.sap/serviceCatalog/kyma-runtime?region=all) and [Regions for the Kyma Environment](https://help.sap.com/docs/btp/sap-business-technology-platform/regions-for-kyma-environment) in SAP Help Portal.

   Optional: You may also change the name of your **Subdomain**. The subdomain will become a part of the URL for accessing applications that you subscribe to from this subaccount. The subdomain must be unique across all subaccounts in the same region. Uppercase and lowercase will not differentiate subdomains.

   ![Provide details for the subaccount](images/22_2_createsub.png)

3. Once created, go to the subaccount and scroll down to **Entitlements**.

>    If you use an SAP BTP trial account, you are already entitled to use the **Kyma runtime** service and can skip the "Entitle your subaccount to use Kyma runtime" step.

Author: Maybe both?

### Entitle your subaccount to use Kyma runtime

> If you use an SAP BTP trial account, you are already entitled to use the **Kyma runtime** service and can skip the "Entitle your subaccount to use Kyma runtime" step. If you use an SAP BTP enterprise account, you are not yet entitled to use the **Kyma runtime** service.

Reviewer: ```suggestion
If you use an SAP BTP trial account, you are already entitled to use the **Kyma runtime** service and can skip this step. If you use an SAP BTP enterprise account, you are not yet entitled to use the **Kyma runtime** service.
```
